dhis2-mobile-program-rules/src/main/java/org/dhis2/mobileProgramRules/RuleEngineExtensions.kt

```diff
@@ -424,12 +424,12 @@ fun getOptions(
         dataElementRepository.uid(dataElementUid).blockingGet()?.optionSet()?.uid()
             ?.let { optionSetUid ->
                 optionRepository.byOptionSetUid().eq(optionSetUid).blockingGet()
-            }?.map { option -> Option(option.name()!!, option.code()!!) } ?: emptyList()
+            }?.map { option -> Option(option.name()!!, option.code() ?: "") } ?: emptyList()
```

Comment:
Could it be that option.name() is also null?

Reply:
well, the server has it as compulsory so in theory it should never be null